discovery-provider/plugins/notifications/src/__tests__/emailRender.test.ts

```diff
@@ -177,28 +190,40 @@ describe('Render email', () => {
       email: 'joey@audius.co',
       frequency: 'daily',
       notifications,
-      dnDb: processor.discoveryDB
+      dnDb: processor.discoveryDB,
+      identityDb: processor.identityDB
     })
     expect(notifHtml).toMatchSnapshot()
   })
 
   test("Render a single Remix Cosign Track Email", async () => {
     await createTracks(processor.discoveryDB, [
-      { track_id: 1, owner_id: 1 },
-      { track_id: 2, owner_id: 2, remix_of: { tracks: [{ parent_track_id: 1 }] } },
+      {
+        track_id: 1,
+        title: "track 1",
+        owner_id: 1,
+        remix_of:
+          { tracks: [{ parent_track_id: 2 }] 
+          }
+      },
+      {
+        track_id: 2,
+        title: "track 2",
+        owner_id: 2
+      },
     ])
     await new Promise(resolve => setTimeout(resolve, 10))
 
     const notifications: AppEmailNotification[] = [
       {
         type: 'cosign',
```

Comment:
to make sure i'm understanding this notification correctly: this is the desired notif for this cosign notification right (where user1's track1 is a remix of user2's track2 and user2 cosigns track1)
<img width="662" alt="image" src="https://user-images.githubusercontent.com/25782182/219320744-e50cc33d-3ff9-4a91-96e2-7547da8c8d1c.png">

Reply:
Yes if track 2 is owned by the user being notified and is the original track - where user 2 remixed track 2 in track 1